hoomd/mpcd/CollisionMethod.h

```diff
@@ -103,9 +107,20 @@ class PYBIND11_EXPORT CollisionMethod : public Autotuned
     uint64_t m_period;        //!< Number of timesteps between collisions
     uint64_t m_next_timestep; //!< Timestep next collision should be performed
 
+    bool m_checked_collision_warnings; //!< True if collision related warnings have been checked
```

Comment:
Make sure to explicitly initialize this variable in the constructor to false, rather than rely on its default behavior.